frontend/README.md

```diff
@@ -1,53 +1,45 @@
-# React Webpack Typescript Starter
-> Minimal starter with hot module replacement (HMR) for rapid development.
+# CoffeeHouse Frontend
 
-* **[React](https://facebook.github.io/react/)** (16.x)
-* **[Webpack](https://webpack.js.org/)** (4.x)
-* **[Typescript](https://www.typescriptlang.org/)** (3.x)
-* **[Hot Module Replacement (HMR)](https://webpack.js.org/concepts/hot-module-replacement/)** using [React Hot Loader](https://github.com/gaearon/react-hot-loader) (4.x)
-* [Babel](http://babeljs.io/) (7.x)
-* [SASS](http://sass-lang.com/)
-* [Jest](https://facebook.github.io/jest/) - Testing framework for React applications
-* Production build script
-* Image loading/minification using [Image Webpack Loader](https://github.com/tcoopman/image-webpack-loader)
-* Typescript compiling using [Awesome Typescript Loader](https://github.com/s-panferov/awesome-typescript-loader) (5.x)
-* Code quality (linting) for Typescript.
+This project uses [React](https://facebook.github.io/react/) and
+[Typescript](https://www.typescriptlang.org/) for the frontend. 
 
-## Installation
-1. Clone/download repo
-2. `yarn install` (or `npm install` for npm)
+Before running any code, one must use `npm install` to install the required dependencies.
 
-## Usage
-**Development**
 
-`yarn run start-dev`
+## Running Locally
+Run the dev server with the command `npm run start-dev`
 
-* Build app continuously (HMR enabled)
-* App served @ `http://localhost:8080`
```

Comment:
Formatting it like this might be a bit easier: To use, visit http://localhost:9000

Reply:
That's a good point! I've changed all references to to ports to follow that style.